src/encoded/static/components/body_map.js

```diff
@@ -4,7 +4,8 @@ import queryString from 'query-string';
 import url from 'url';
```

Comment:
Bug
- Go to- https://encd-5693-1a74df763-emma.demo.encodedcc.org/reference-epigenome-matrix/?type=Experiment&related_series.@type=ReferenceEpigenome&replicates.library.biosample.donor.organism.scientific_name=Mus+musculus

- Click on expand the SVG

- Click on large intestine image on the mouse

- You get a missing page error.

Note- You don't get the error if you click on a different organ and then the large intestine. Other times, I get similar errors when I click on other items.

Reply:
This is sort of a bug and also sort of not a bug. A number of the organ slims on the diagram do not have associated data. It's possible we should hide / disable organ slims that do not have data. Do you mind commenting about this on the ticket? It's probably a question for Bonita and Ben, what to do about this.

Reply:
This update took some fixing. We did want to disable buttons / svg paths for organs and systems which resulted in no data. The bulk of the changes address this.